xml/cl.xml

```diff
@@ -7409,6 +7410,12 @@ server's OpenCL/api-docs repository.
                <enum name="CL_COMMAND_BUFFER_STATE_RECORDING_KHR"/>
                <enum name="CL_COMMAND_BUFFER_STATE_EXECUTABLE_KHR"/>
             </require>
+            <require comment="cl_command_buffer_state_khr">
+                <enum name="CL_COMMAND_BUFFER_STATE_FINALIZED_KHR"/>
+            </require>
+            <require comment="cl_command_buffer_state_khr">
+                <enum name="CL_COMMAND_BUFFER_STATE_FINALIZED_KHR"/>
+            </require>
```

Comment:
This is entry is duplicated, but separate from that I think we should be defining this under the `cl_khr_command_buffer_mutable_dispach`  `<extension>` tag and bumping it's version number to 0.9.5, rather than adding it to  `cl_khr_command_buffer` (as we're defining `CL_COMMAND_BUFFER_STATE_FINALIZED_KHR` in `api/cl_khr_command_buffer_mutable_dispatch.asciidoc`)